channelnotifier/channelnotifier.go

```diff
@@ -73,7 +73,10 @@ func (c *ChannelNotifier) Stop() {
 }
 
 // SubscribeChannelEvents returns a subscribe.Client that will receive updates
-// any time the Server is made aware of a new event.
+// any time the Server is made aware of a new event. The subscription provides
+// channel events from the point of subscription onwards. TODO: update  to allow
+// subscriptions to specify a block height from which we would like to subscribe
+// to events.
```

Comment:
I think, but not sure if we really do this always, that we make the TODO stand out a bit more by adding an empty comment line above it.

Reply:
Yes, add it on a separate line, and with your handle, like this:
`TODO(carlaKC): update to....`